dashboard/README.md

```diff
@@ -32,11 +32,24 @@ Start a RisingWave cluster, create some tables and materialized views for testin
 For example:
 
 ```bash
-./risedev d
+./risedev d full
```

Comment:
Crucial metadata `parentIds` is missing from fragment metadata when I use `risedev d`. So use `risedev d full` instead.

Reply:
Why the deployment can affect metadata?

Reply:
I'm not sure either. Still looking into it. You can see the graph looks like that without it, since we can't draw edges between the nodes as there's no parentIds.

<img width="130" alt="Screenshot 2024-09-19 at 6 01 13 PM" src="https://github.com/user-attachments/assets/e85f18a3-079c-43c3-ba9d-d2ce06d21455">

Reply:
Oh I suppose it's because this branch still uses etcd for `risedev d full`. The etcd backend in meta still has the parent dependencies stored as part of metadata for the relation.

But the sqlbackend does not, which is what `risedev d` uses.

I will update the implementation.

Reply:
Fixed in https://github.com/risingwavelabs/risingwave/pull/18280/commits/abb2a72d6b765a1e5d6a855c7952bbf2fe90db5b